posix.mak

```diff
@@ -340,12 +340,199 @@ endif
 $(addprefix $(ROOT)/unittest/,$(DISABLED_TESTS)) :
 	@echo Testing $@ - disabled
 
+# Module-specific compiler switches (needed for the -dip1000 transition)
+#etc/
+aa[etc.c.curl]=-dip1000
+aa[etc.c.sqlite3]=-dip1000
+aa[etc.c.zlib]=-dip1000
+
+aa[etc.c.odbc.sql]=-dip1000
+aa[etc.c.odbc.sqlext]=-dip1000
+aa[etc.c.odbc.sqltypes]=-dip1000
+aa[etc.c.odbc.sqlucode]=-dip1000
+
+#std/
+aa[std.array]=-dip1000
+aa[std.ascii]=-dip1000
+aa[std.base64]=-dip1000
+aa[std.bigint]=-dip1000
+aa[std.bitmanip]=-dip1000 # PR 6174
```

Comment:
Walter is a huge fan of clickable links

Reply:
I'm a fan of efficiency (including clickable links; sorry, in a hurry I sometimes forget them).
In this case I don't know if they work from there, but I'll update to URLs.